packages/web-react/README.md

```diff
@@ -69,6 +69,15 @@ are doing so at your own risk.
 
 Please consult additional styling with [web package documentation][web-pkg-rebrand].
 
+## Development
+
+- `% yarn storybook` starts development server with Storybook
+
+## Testing
+
+- `% cd <your-local-path>/spirit-design-system/packages/web-react`
+- `% yarn test` or `% yarn test:unit`
```

Comment:
This is misleading. Do you want to deal with unit testing (running jest tests) or testing of the entire package (lint, format, unit testing, types)?